src/modules/GroupChannel/components/Message/MessageView.tsx

```diff
@@ -146,7 +147,7 @@ const MessageView = (props: MessageViewProps) => {
     animatedMessageId,
     onMessageAnimated,
     usedInLegacy = true,
-  } = props;
+  } = deleteNullish(props);
```

Comment:
@liamcho @HoonBaek @AhyoungRyu

When `renderXXX` props are mixed with regular props like this, would it be better to separate them?
Have concerned about potential side effects in the logic when customers pass `null` and it changes to `undefined`. (For example, when business logic checks conditions using `=== null`.)

```tsx
// AS-IS
const {
  propA,
  propB,
  renderPropC = () => <DefaultComponent />
} = deleteNullish(props);

// TO-BE
const { propA, propB } = props;
const { renderPropC = () => <DefaultComponent /> } = deleteNullish(props);
```

Reply:
Its the same no? propA, propB are references so they could be removed after deleteNullish(props)

Reply:
If we are distinguishing between undefined and null for propA and propB in our business logic, the outcome will differ. (The customer passed null, but it is handled as undefined)